src/app/error.tsx

```
@@ -0,0 +1,84 @@
+"use client"
+
+import { useEffect } from "react"
+import { Card, CardContent, CardDescription, CardHeader, CardTitle } from "~/components/ui/card"
+import { Button } from "~/components/ui/button"
+import { AlertTriangle, RefreshCw, Home, ArrowLeft } from "lucide-react"
+import Link from "next/link"
+
+export default function Error({
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)